src/rules/prefer-expect-assertions.ts

```diff
@@ -61,8 +74,17 @@ export default createRule({
       recommended: false,
     },
     messages: {
+      hasAssertionsTakesNoArguments:
+        '`expect.hasAssertions` expects no arguments',
+      assertionsRequiresOneArgument:
+        '`expect.assertions` excepts a single argument of type number',
+      assertionsRequiresNumberArgument: 'This argument should be a number',
       haveExpectAssertions:
         'Every test should have either `expect.assertions(<number of assertions>)` or `expect.hasAssertions()` as its first expression',
+      suggestAddingHasAssertions: 'Add `expect.hasAssertions()`',
+      suggestAddingAssertions:
+        'Add `expect.assertions(<number of assertions>)`',
+      suggestRemovingExtraArguments: 'Remove extra arguments',
```

Comment:
@SimenB interested in hearing your thoughts on doing this as a suggestion vs autofix.

I did initially have it just fixing, but that causes a knock on effect by requiring setting `fixable` which gets pulled into the README, which is ok but at the same time I'm not entirely happy about because only 1 specific part of this rule is actually "fixable".

I'm reasonably happy with using suggestions, but interested in your thoughts on if this is valuable as a suggestion/fix/at-all, as can apply this to other rules too, such as `valid-expect`, `valid-describe`, etc

Reply:
can we have some "partially fixable" in the readme?